frontend/package.json

```
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "remix vite:build",
     "deploy": "pnpm run build && wrangler pages deploy",
-    "dev": "remix vite:dev",
+    "dev": "NODE_TLS_REJECT_UNAUTHORIZED=0 remix vite:dev",
```

なんのための環境変数かをコメントで残しておきたい

```suggestion
// 自己署名証明書の検証をスキップ
"dev": "NODE_TLS_REJECT_UNAUTHORIZED=0 remix vite:dev",
```

---

おk

---

package.jsonにコメントかくとvscodeがエラー吐くんやが

---

じゃあ諦めよう